src/protocols/rdp/compat/winpr-stream.h

```diff
@@ -60,6 +60,7 @@
 #define Stream_Buffer                  stream_get_head
 #define Stream_Pointer                 stream_get_tail
 #define Stream_Length                  stream_get_size
+#define Stream_Copy                    stream_copy
```

Comment:
Is this still needed as of the latest changes?

Reply:
Nope.

Reply:
Removed.